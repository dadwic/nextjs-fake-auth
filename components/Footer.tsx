import React from "react";
import Box from "@mui/material/Box";
import Link from "@mui/material/Link";
import Typography from "@mui/material/Typography";

function Copyright() {
  return (
    <Typography variant="body2" color="text.secondary">
      {`Copyright © ${new Date().getFullYear()}`}
    </Typography>
  );
}

export default function Footer() {
  return (
    <Box
      component="footer"
      sx={{
        mt: 5,
      }}
    >
      <Typography variant="body1">
        Made with ❤️ by{" "}
        <Link
          color="inherit"
          target="_blank"
          href="https://github.com/dadwic"
          rel="noopener noreferrer"
        >
          @dadwic
        </Link>
      </Typography>
      <Copyright />
    </Box>
  );
}
