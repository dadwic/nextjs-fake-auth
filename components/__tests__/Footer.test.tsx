import { render } from "@testing-library/react";
import Footer from "../Footer";

it("renders unchanged", () => {
  const { container } = render(<Footer />);
  expect(container).toMatchSnapshot();
});
