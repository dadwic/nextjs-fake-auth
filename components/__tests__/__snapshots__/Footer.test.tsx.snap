// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders unchanged 1`] = `
<div>
  <footer
    class="MuiBox-root css-y1gt6f"
  >
    <p
      class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
    >
      Made with ❤️ by
       
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1cghntr-MuiTypography-root-MuiLink-root"
        href="https://github.com/dadwic"
        rel="noopener noreferrer"
        target="_blank"
      >
        @dadwic
      </a>
    </p>
    <p
      class="MuiTypography-root MuiTypography-body2 css-r40f8v-MuiTypography-root"
    >
      Copyright © 2022
    </p>
  </footer>
</div>
`;
